app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -658,6 +658,10 @@ ShortFormApplicationController = (
   $scope.listingIsSale = ->
     ShortFormApplicationService.listingIsSale()
 
+  $scope.checklistSection = ->
```

Comment:
same here, just wondering what the purpose of this function is.

Reply:
It would should homebuyers guide. I don't think we are doing it now though and in the story it was said that there will be two different pages for sale and rental listing later on, so it's a temporary thing I believe. I can also modify it to work for both, default 'sale' and preference 'sale'

Reply:
can we make it a bit clearer what the function is doing?